src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustIndividualValidator.java

```diff
@@ -43,35 +43,29 @@ public TrustIndividualValidator(AddressDtoValidator addressDtoValidator,
     }
 
     public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, String loggingContext) {
+        String type = null;
```

Comment:
Why declare type up here, why not on line 55?

Reply:
thanks @ch-tstrickland I have corrected it.